src/internalUtils.js

```diff
@@ -0,0 +1,15 @@
+import { vec3 } from 'gl-matrix'
+import { computeBoundsFromPoints } from 'vtk.js/Sources/Common/Core/Math'
+import vtkBoundingBox from 'vtk.js/Sources/Common/DataModel/BoundingBox'
+
+export const transformBounds = (transformingMat4, bounds) => {
```

Comment:
To keep semantically meaningful modules / file names, please put this into a `transformBounds` file / module.